tests/test_broadcast.py

```diff
@@ -22,16 +23,18 @@ async def test_redis():
             assert event.channel == "chatroom"
             assert event.message == "hello"
 
-## pulsar test 
+
 @pytest.mark.asyncio
 async def test_pulsar():
     async with Broadcast("pulsar://localhost:6650") as broadcast:
         async with broadcast.subscribe("chatroom") as subscriber:
+            await asyncio.sleep(2)  # Increase this value
```

Comment:
Why is this sleep needed?

Reply:
can't fully remember... buh i knew pulsar take time to boot up...